src/app/GitUI/UserControls/RevisionGrid/QuickSearchProvider.cs

```diff
@@ -49,6 +49,9 @@ public QuickSearchProvider(RevisionDataGridView gridView, Func<string> getWorkin
 
         public void OnKeyPress(KeyPressEventArgs e)
         {
+            // SYN char is issued when pasting
+            const char SynChar = (char)22;
```

Comment:
👍 for constant - but with speaking name, please
```suggestion
            // Ctrl+A to Ctrl+Z have codes 1..26
            const char ctrlVChar = (char)('V' - 'A' + 1);
```
Delphi case is used for public constants only.

Reply:
I've extended your comment with clarification for `Cltr+V being 22` I think it's important for this value to be mentioned.

Reply:
Haha I found your :confused: reaction funny 😅 my point was that it's a long stretch from `== 22` to `'V' - 'A' + 1` and you do see `22` when debugging thus I thought it was beneficial to have it "visible" in the code at least somewhere. Like I've mentioned before, when I googled for `22 copy keycode` I've found no char math being performed hense my though that a comment should provide some googlable context using keywords that are present in posts on this topic for future devs who'd stumble upon this code.